config/hiveadmission/konnectivity-agent.yaml

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
Per @csrwng, if we use a URL rather than a service ref in my APIService, we shouldn't need this konnectivity agent. He also asserts we should be able to do this without requiring any additional authz, which should mean the existing cert we're copying in (for authn) should be adequate.
Presumably we would also not need the Endpoints or the mirror Service in the HCP with this setup.